sky/packages/flutter_services/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: flutter_services
+version: 0.0.1
+author: Flutter Authors <flutter-dev@googlegroups.com>
+description: Services exported from the Flutter engine
+homepage: http://flutter.io
+dependencies:
+  sky_services:
+    path:
+      ../sky_services
```

Comment:
That dependency is pinned via mojo_sdk, which the sky_services/pubspec.yaml depends on. I could express an (exact) dependency on mojo_sdk here as well but it'd have to exactly sky_services/pubspec.yaml exactly. I'm planning to make flutter_services not have any imports from the mojo_services package soon so the problem goes away.